### PR TITLE
feat: bump components version to allow searching descriptions on Selects

### PR DESCRIPTION
**Pull request by felixxiatransferwise**
## 🖼 Context

Right now in dynamic flows, we are only searching the label for `Select` options and ignoring `note` and `secondary`, however these should be searchable. 

## 🚀 Changes

 
- Bump components version
- Various refactor improvements

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+};
+
+const mapIcon = (icon) => (icon ? getCurrencyFlag(icon.name) : null);
+
+const getSearchStrings = (schemaDescription) => {
+  if (schemaDescription) {
+    return { searchStrings: [schemaDescription] };
+  }
+  return undefined;
+};
```
Comment (felixxiatransferwise) [inline packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js:22]: The main addition in this PR, rest of the changes are refactors and slight improvements.

File: packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { getCurrencyFlag } from './availableCurrencyFlags';
+
+export const mapConstSchemaToOption = (schema) => {
+  return {
+    label: schema.title,
+    value: !isUndefined(schema.const) ? schema.const : schema.enum[0],
+    ...getOptionDescription(schema.title, schema.description),
+    ...mapIcon(schema.icon),
+    ...getDisabled(schema.disabled),
+    ...getSearchStrings(schema.description),
```
Comment (felixxiatransferwise) [inline packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js:11]: This is so we avoid getting `key: undefined` in our option object.

File: packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js (hunk 1 above)
Comment (StevePole) [reply packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js:22]: Does it search the title automatically?

Comment (felixxiatransferwise) [reply packages/dynamic-flows/src/jsonSchemaForm/schemaFormControl/optionMapper.js:22]: Yes

**Review by StevePole — APPROVED** [review]
👍

**Pull request merged**